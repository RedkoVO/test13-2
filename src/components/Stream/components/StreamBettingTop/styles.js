export default () => ({
  root: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    padding: '0 50px',
    width: '810px',
    background: 'rgba(0, 0, 0, 0.20)',
    height: '100px',
    borderRadius: '15px 15px 0 0'
  },
  streamLeftTeam: {
    display: 'flex',
    color: '#fff'
  },
  streamLeftLogo: {
    width: '50px',
    paddingRight: '6px'
  },
  streamTeamName: {
    display: 'flex',
    alignItems: 'center',
    fontSize: '40px',
    fontWeight: 'bold'
  },
  streamCountry: {
    paddingLeft: '40px',
    fontSize: '15px',
    fontWeight: 'bold',
    textAlign: 'center'
  },
  streamFlag: {
    paddingRight: '2px',
    width: '17px'
  },
  streamCoef: {
    display: 'flex',
    alignItems: 'center',
    color: '#fff',
    fontSize: '20px',
    fontWeight: 'bold'
  },
  streamVersus: {
    padding: '0 15px'
  },
  streamCoefItem: {
    padding: '8px 15px',
    fontSize: '30px',
    background: 'rgba(0, 0, 0, 0.15)',
    borderRadius: '15px'
  },
  streamRightTeam: {
    display: 'flex',
    position: 'relative',
    color: '#fff'
  },
  streamRightLogo: {
    width: '30px',
    paddingRight: '6px'
  },
  streamLive: {
    position: 'absolute',
    top: 0,
    right: '-60px',
    fontSize: '15px',
    fontWeight: 'bold',

    '&:before': {
      content: `""`,
      position: 'absolute',
      top: '6px',
      left: '-11px',
      width: '7px',
      height: '7px',
      backgroundColor: 'red',
      borderRadius: '50px'
    }
  },
  streamEvent: {
    marginLeft: '125px',
    color: '#fff',
    fontSize: '7px',
    fontWeight: 'bold',
    textAlign: 'center',

    '& img': {
      display: 'block',
      width: '50px'
    }
  }
})
