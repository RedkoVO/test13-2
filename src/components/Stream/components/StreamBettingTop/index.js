import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import LeftTeamLogo from '../../../../assets/images/leftTeamLogo.png'
import LeftTeamFlag from '../../../../assets/images/leftTeamFlag.png'
import RightTeamLogo from '../../../../assets/images/rightTeamLogo.png'
import RightTeamFlag from '../../../../assets/images/rightTeamFlag.png'
import StreamEvent from '../../../../assets/images/streamEvent.png'

import styles from './styles'

const StreamBettingTop = ({ classes }) => (
  <div className={classes.root}>
    <div className={classes.streamLeftTeam}>
      <div>
        <div className={classes.streamTeamName}>
          <img src={LeftTeamLogo} className={classes.streamLeftLogo} alt="" />
          NRG
        </div>
        <div className={classes.streamCountry}>
          <img src={LeftTeamFlag} className={classes.streamFlag} alt="" />
          France
        </div>
      </div>
    </div>

    <div className={classes.streamCoef}>
      <div className={classes.streamCoefItem}>6.17</div>
      <div className={classes.streamVersus}>vs</div>
      <div className={classes.streamCoefItem}>1.25</div>
    </div>

    <div className={classes.streamRightTeam}>
      <div className={classes.streamRightTeam}>
        <div>
          <div className={classes.streamTeamName}>
            <img src={RightTeamLogo} className={classes.streamRightLogo} alt="" />
            LDLC
          </div>
          <div className={classes.streamCountry}>
            <img src={RightTeamFlag} className={classes.streamFlag} alt="" />
            Spain
          </div>
        </div>
      </div>
      <div className={classes.streamLive}>Live</div>
    </div>

    <div className={classes.streamEvent}>
      Event:
      <img src={StreamEvent} alt="" />
    </div>
  </div>
)

StreamBettingTop.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(StreamBettingTop)
