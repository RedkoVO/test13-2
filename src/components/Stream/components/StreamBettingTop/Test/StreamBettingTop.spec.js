import * as React from 'react'
import renderer from 'react-test-renderer'

import StreamBettingTop from '../'

it('renders correctly', () => {
  const tree = renderer.create(<StreamBettingTop />).toJSON()
  expect(tree).toMatchSnapshot()
})
