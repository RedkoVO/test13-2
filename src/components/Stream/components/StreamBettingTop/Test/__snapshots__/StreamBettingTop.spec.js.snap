// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StreamBettingTop-root-1"
>
  <div
    className="StreamBettingTop-streamLeftTeam-2"
  >
    <div>
      <div
        className="StreamBettingTop-streamTeamName-4"
      >
        <img
          alt=""
          className="StreamBettingTop-streamLeftLogo-3"
          src="leftTeamLogo.png"
        />
        NRG
      </div>
      <div
        className="StreamBettingTop-streamCountry-5"
      >
        <img
          alt=""
          className="StreamBettingTop-streamFlag-6"
          src="leftTeamFlag.png"
        />
        France
      </div>
    </div>
  </div>
  <div
    className="StreamBettingTop-streamCoef-7"
  >
    <div
      className="StreamBettingTop-streamCoefItem-9"
    >
      6.17
    </div>
    <div
      className="StreamBettingTop-streamVersus-8"
    >
      vs
    </div>
    <div
      className="StreamBettingTop-streamCoefItem-9"
    >
      1.25
    </div>
  </div>
  <div
    className="StreamBettingTop-streamRightTeam-10"
  >
    <div
      className="StreamBettingTop-streamRightTeam-10"
    >
      <div>
        <div
          className="StreamBettingTop-streamTeamName-4"
        >
          <img
            alt=""
            className="StreamBettingTop-streamRightLogo-11"
            src="rightTeamLogo.png"
          />
          LDLC
        </div>
        <div
          className="StreamBettingTop-streamCountry-5"
        >
          <img
            alt=""
            className="StreamBettingTop-streamFlag-6"
            src="rightTeamFlag.png"
          />
          Spain
        </div>
      </div>
    </div>
    <div
      className="StreamBettingTop-streamLive-12"
    >
      Live
    </div>
  </div>
  <div
    className="StreamBettingTop-streamEvent-13"
  >
    Event:
    <img
      alt=""
      src="streamEvent.png"
    />
  </div>
</div>
`;
